### PR TITLE
add support for laravel 7

### PR DESCRIPTION
**Pull request by JulianMar**
bump laravel version
bump phpunit version
bump php version, to use the same as laravel

drop support for laravel 5.5 - 5.7

### CODE REVIEW

**Comment by RatkoR** [comment]
> drop support for laravel 5.5 - 5.7

Do we need to do this?

**Comment by JulianMar** [comment]
actually I am not quite sure.
I did it because Spatie dropped the support too.

I checked the laravel [support policy](https://laravel.com/docs/7.x/releases#support-policy) looks like I will add it again

**Comment by JulianMar** [comment]
@RatkoR I updated the composer file.
During that I noticed if we have a .lock file, we cannot gurantee the correct installation of the library. It would just install the newest dependencies, so I removed it

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ DB servers a lot.
 
 ### Installation
 
-Laravel 5.5 & 6 is supported. Version tagged with 9.* (and higher) require **php 7.2**!
+Laravel 5.8, 6, 7 are supported. Version tagged with 9.* (and higher) require **php 7.2.5**!
```
Comment (RatkoR) [inline README.md]: I think you forgot to change back `5.8` to `5.5` (like it was before).

Comment (JulianMar) [reply README.md]: yes, I changed it now :)

**Comment by RatkoR** [comment]
Thx, merged!

**Pull request merged**